app/assets/stylesheets/_billables.scss

```diff
@@ -53,6 +53,19 @@
   }
 }
 
+.unbill{
+     background:none!important;
+     border:none;
+     padding:0;
```

Comment:
Line should be indented 2 spaces, but was indented 5 spaces<br>Properties should be ordered background, border, color, cursor, font, min-width, padding<br>Colon after property should be followed by one space